src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -38,6 +38,7 @@
     public boolean respawnOnIsland = true;
     public boolean obsidianBucket = true;
     public boolean patreonMessage = true;
+    public boolean disableIslandMenu = false;
```

Comment:
also I think this should be the opposite
public boolean islandMenu = true

Reply:
so instead of disabling it have the boolean be to enable it and default to true

Reply:
Yeah, I was thinking the same, no idea why I didn't do it